tests/basic/storage_pool_test.go

```diff
@@ -6880,7 +6880,7 @@ var _ = Describe("{DriveAddPXDown}", func() {
 
 		// Add Drive on the Node [ PTX-15856 ]
 		err = AddCloudDrive(*nodeDetail, -1)
-		log.FailOnError(err, "adding new pool on the node failed?")
+		dash.VerifyFatal(err != nil, true, "adding new pool on the node failed?")
```

Comment:
The expected behaviour is the test case should pass if an error is thrown while pool expand is initiated on a node where px is down. So 
`dash.VerifyFatal(err, nil, "adding new pool on the node failed?")` 
would be expecting no error and if error is seen it will fail the test which is not the intention of the test.
`dash.VerifyFatal(err!=nil, true, "adding new pool on the node failed?")` would be the right way to do it. Maybe I can improve the logic to show the error as well

Reply:
Right. My bad.